doc/rst/language/spec.rst

```diff
@@ -1652,25 +1671,31 @@ in :ref:`Constants`. Ref variables are discussed in
 :ref:`Ref_Variables`.
 
 The ``type-part`` of a variable declaration specifies the type of the
-variable. It is optional if the ``initialization-part`` is specified. If
-the ``type-part`` is omitted, the type of the variable is inferred using
-local type inference described in :ref:`Local_Type_Inference`.
-If the ``type-part`` refers to a generic type, then an
-``initialization-part`` is required and will be used to determine the
-type of the variable. In this event, the compiler will fail with an
-error if the ``initialization-part`` is not coercible to an
-instantiation of the generic type.
+variable. It is optional.
 
 The ``initialization-part`` of a variable declaration specifies an
-initial expression to assign to the variable. If the
-``initialization-part`` is omitted, the ``type-part`` must be present,
-and the variable is initialized to the default value of its type as
-described in :ref:`Default_Values_For_Types`.
-
-If the ``no-initialization-part`` is present, the variable declaration
-does not initialize the variable to any value, as described
-in :ref:`Noinit_Capability`. The result of any read of an
-uninitialized variable is undefined until that variable is written.
+initial expression to store into the variable. It is also optional. When
+present, it forms the initialization expression for the variable.
+
+If the ``initialization-part`` is omitted, the compiler will consider if
+split initialization can be applied to this variable as described in
+:ref:`Split_Initialization`. If split initialization can be applied, the
+compiler will identify a later assignment statement and the
+right-hand-side of that statement will form the initialization
+expression. If the ``initialization-part`` is omitted and split
+initialization cannot be applied, then the variable will need to be
+initialized to a default value. Only `var` and `const` variable
+declarations can be initialized to a default value. Not all types have a
+default value. Default values are described in
+:ref:`Default_Values_For_Types`.
+
```

Comment:
Adjust the following paragraph:
```
If the ``type-part`` is omitted or refers to a generic type,
the initialization expression is required and will ...
```
Then, add `concrete` in `determine the concrete type of the variable`.

Continuing from there:
```
If the ``type-part`` is present, the initialization expression
must be coercible to the specified type or, if it is generic,
to its instantiation.